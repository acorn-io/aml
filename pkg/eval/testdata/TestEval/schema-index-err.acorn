 schema {
	"items": [
		{
			aPositionNumber:    number > 0
			optionalNum?:       number
			optionalBool?:      bool
			optionalString?:    string
			anExplicitConstant: string == "value"
			aRegex:             string =~ "str.*"
			aNumberRange:       number > 0 && number < 10 || default 1
			defaultedString:    default "hi"
			match "field.*":    string
		},
	]
}

{
	items: [{aPositionNumber: 3
					anExplicitConstant: "value"
					aRegex:             "strip"
					aNumberRange:       9
					optionalNum:        1_00_00
					optionalBool:       true
					optionalString:     "test123"
	},
		{aPositionNumber: -4
					anExplicitConstant: "value"
					aRegex:             "strip"
					aNumberRange:       9
					optionalNum:        1_00_00
					optionalBool:       true
					optionalString:     "test123"
		},
	]
}
