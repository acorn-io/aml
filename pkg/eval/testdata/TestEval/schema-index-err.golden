"schema violation key items.aPositionNumber: constraint [value > 0] is not true [path items[1].aPositionNumber] [schema path Foo.items[0]]: schema-index-err.acorn:17:4 (12:10<-2:11<-2:2<-17:4)"
